src/SeDuMi.jl

```diff
@@ -22,6 +22,16 @@ mutable struct Cone
     ycomplex::Vector{Float64} #list of constraints on A*x that should also act on the imaginary part
     xcomplex::Vector{Float64} #list of components of f,l,q,r blocks allowed to be complex
 end
+function Cone(
+    f::Real,
+    l::Real,
+    q::Vector,
+    r::Vector,
+    s::Vector,
+    scomplex::Vector,
+)
+    return Cone(f, l, q, r, s, scomplex, Float64[], Float64[])
+end
 function Cone(f::Real, l::Real, q::Vector, r::Vector, s::Vector)
```

Comment:
Instead of adding a new method above, you could just add a default argument to this one:
```suggestion
function Cone(f::Real, l::Real, q::Vector, r::Vector, s::Vector, scomplex::Vector = Float64[])
```